packages/overture-schema-transportation-theme/src/overture/schema/transportation/models.py

```diff
@@ -48,6 +50,12 @@
 Width = Annotated[float, Field(gt=0)]
 
 
+def _connector_type() -> type[Feature]:
+    from overture.schema.transportation.connector import Connector
```

Comment:
```suggestion
    from .connector import Connector
```

Reply:
Just updated the PR with a commit that applies this change.